src/index.js

```diff
@@ -22,7 +22,6 @@ export { IndexedList } from './core/indexed-list.js';
 export { Tags } from './core/tags.js';
 export { Timer, now } from './core/time.js';
 export { URI, createURI } from './core/uri.js';
-export { RefCountedObject } from './core/ref-counted-object.js';
```

Comment:
so a class does not need to be added here if it's private?

Reply:
Correct. The `index.js` module defines the classes/function/properties that are exported to the `pc` namespace.